cicle.run_deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+
+# TODO: region should be ENV determined
```

Comment:
create issue here

Reply:
I'll drop the whole file, last version does not use it anymore